src/content/components/ReleaseReport/ReleaseReport.js

```diff
@@ -54,47 +40,187 @@ export class ReleaseReport extends React.PureComponent {
-                        <a href={OPEN_BUG_URL + bug.id}>{bug.summary}</a>
-                      </li>
-                    ))}
-                  </ul>
-                </div>
-              );
-            })
-        ) : (
-          <Loader />
-        )}
+class ReleaseDatesTab extends React.PureComponent {
+  getStartDate(milestone) {
+    let iterationDate = ITERATION_LOOKUP.byVersionString[milestone];
+    let milestoneDate = new Date(iterationDate.startDate);
+
+    return milestoneDate;
+  }
+
+  getMilestoneDate(milestone) {
+    const iteration = getIteration();
+    const release = iteration.number.split(".")[0];
+    const nextRelease = parseInt(release) + 1;
+    let milestoneDate = "";
+
+    if (milestone === "Merge Day") {
+      milestone = `${nextRelease}.1`;
+      // use the converted milestone to run lookup
+      milestoneDate = this.getStartDate(milestone);
+    } else if (milestone === "Release Day") {
+      milestone = `${nextRelease}.1`;
+      milestoneDate = this.getStartDate(milestone);
+
+      milestoneDate.setDate(milestoneDate.getDate() + 1);
+    } else {
+      // use the milestone to run lookup
+      milestoneDate = this.getStartDate(milestone);
+    }
+
+    return milestoneDate;
+  }
+
+  render() {
+    return (
+      <div>
+        <div className={styles.summary}>
+          <p>
+            These are projected dates for the major milestones in {PROJECT_NAME}{" "}
+            {release}.
+          </p>
+          <p>
+            See <a href={TIMELINE_DOC}>this document</a> for more information.
+          </p>
+        </div>
+        <div className={styles.table}>
+          <table>
+            <tbody>
+              <tr>
+                <th>Milestone</th>
+                <th>Day</th>
+                <th>Date</th>
+              </tr>
+              {this.props.milestones.map(milestone => {
+                const date = this.getMilestoneDate(milestone);
+                return (
+                  <tr key={milestone}>
+                    <td>
+                      <strong>{milestone}</strong>
+                    </td>
+                    <td>
+                      <time>{date.toDateString()}</time>
+                    </td>
+                    <td>
+                      <time>{new Intl.DateTimeFormat().format(date)}</time>
+                    </td>
+                  </tr>
+                );
+              })}
+            </tbody>
+          </table>
+        </div>
       </div>
     );
   }
 }
+
+export class ReleaseReport extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      bugs: [],
+      loaded: false,
+      milestones: [
+        `${release}.1`,
+        `${release}.2`,
+        `${release}.3`,
+        `${release}.4`,
+        "Merge Day",
+        "Release Day",
+      ],
+    };
+  }
+
+  async componentWillMount() {
+    this.setState({ loaded: false });
+    const result = await runQuery({
```

Comment:
We need to actually update this query, it's not the most accurate anymore. We can do as a follow-up though